tests/test.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Please could you change the extension to `.yaml`? I kind of don't like the shorter `.yml`. It's like you would use `.jon` instead of `.json` ;o)